src/Data/PlaceCal/Events.elm

```diff
@@ -91,6 +91,15 @@ eventsFromDate eventsList fromDate =
         eventsList
 
 
+eventsAfterDate : List Event -> Time.Posix -> List Event
+eventsAfterDate eventsList fromDate =
```

Comment:
I think because this is scoped under `Events`, we can do this:

```suggestion
afterDate : List Event -> Time.Posix -> List Event
afterDate eventsList fromDate =
```

And then call it with `Events.afterDate` instead of the repetitive `Events.eventsAfterDate`. Feels more like Elm?